test/unit/component/parser/json/BpmnJsonParser.startEvent.test.ts

```diff
@@ -146,4 +146,63 @@ describe('parse bpmn as json for start event', () => {
       boundsHeight: 46,
     });
   });
+
+  it('json containing one process with an array of start events, some are not NONE event', () => {
+    const json = `{
+  "definitions": {
+    "process": {
+      "startEvent": [
+        { "id": "event_id_0", "name": "none start event" },
+        { "id": "event_id_1", "conditionalEventDefinition": {} },
+        { "id": "event_id_2", "messageEventDefinition": {} },
+        { "id": "event_id_3", "signalEventDefinition": {} },
+        { "id": "event_id_4", "timerEventDefinition": {} }
+      ]
+    },
+    "BPMNDiagram": {
+      "name": "process 0",
+      "BPMNPlane": {
+        "BPMNShape": [
+          {
+            "id": "shape_startEvent_id_0", "bpmnElement": "event_id_0",
+            "Bounds": { "x": 362, "y": 232, "width": 36, "height": 45 }
+          },
+          {
+            "id": "shape_startEvent_id_1", "bpmnElement": "event_id_1",
+            "Bounds": { "x": 362, "y": 332, "width": 36, "height": 45 }
+          },
+          {
+            "id": "shape_startEvent_id_2", "bpmnElement": "event_id_2",
+            "Bounds": { "x": 362, "y": 432, "width": 36, "height": 45 }
+          },
+          {
+            "id": "shape_startEvent_id_3", "bpmnElement": "event_id_3",
+            "Bounds": { "x": 362, "y": 532, "width": 36, "height": 45 }
+          },
+          {
+            "id": "shape_startEvent_id_4", "bpmnElement": "event_id_4",
+            "Bounds": { "x": 362, "y": 632, "width": 36, "height": 45 }
+          }
+        ]
+      }
+    }
+  }
+}`;
+
+    const model = parseJson(json);
+
+    const noneStartEvents = model.flowNodes.filter(shape => shape.bpmnElement.kind == ShapeBpmnElementKind.EVENT_START);
```

Comment:
I think there is a duplication with **verifyShape**, as we have only one flownode in the model & juste start events in the json ;)

Reply:
Yes, you are totally right.
Implemented with 76738ec0